src/sentry/event_manager.py

```diff
@@ -542,11 +543,15 @@ def _get_event_user(self, project, data):
         if not euser.tag_value:
             return
 
-        try:
-            with transaction.atomic():
-                euser.save()
-        except IntegrityError:
-            pass
+        cache_key = 'euser:{}'.format(md5(euser.tag_value).hexdigest())
```

Comment:
I assume it should match the unique_together index or we're going to have issues here from my understanding.